x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/action/FollowStatsAction.java

```diff
@@ -92,7 +93,10 @@ private static Iterator<ToXContent> innerToXContentChunked(Map<String, Map<Integ
                     taskResponsesByIndex.entrySet().iterator(),
                     indexEntry -> Iterators.concat(
                         Iterators.<ToXContent>single(
-                            (builder, params) -> builder.startObject().field("index", indexEntry.getKey()).startArray("shards")
+                            (builder, params) -> builder.startObject()
+                                .field("index", indexEntry.getKey())
+                                .field("follower_to_leader_lagging_ops_count", calcFollowerToLeaderLaggingOps(indexEntry.getValue()))
```

Comment:
I am surprised that this is per index, not per shard. Any reason why it deviates from the rest of the stats?

Reply:
I think the reason is be an *accumulative* stat across all shards per index. main motto is stated by parent issue:
>These should be exposed as simple user-friendly metrics without going through extra mental arithmetic.